qualtran/_infra/data_types.py

```diff
@@ -184,28 +172,46 @@ def num_qubits(self):
 
 
 @attrs.frozen
-class QFixedPoint(QDType):
-    """Fixed point type to represent real numbers.
+class QFxp(QDType):
+    r"""Fixed point type to represent real numbers.
 
-    The integer part of the float and the fractional part (the part after the
-    decimal plase) can be chosen to have different bitsizes. An additional sign bit is assumed.
+    A real number can be approximately represented in fixed point using num_int
+    bits for the integer part and num_frac bits for the fractional part. If the
+    real number is signed we require an additional bit to store the sign (0 for
+    +, 1 for -). In total there are bitsize = (n_sign + num_int + num_frac) bits used
+    to represent the number. E.g. n_sign = 0 if bitsize = 8, num_frac = 6, signed = False,
+    then $\pi$ \approx 3.140625 = 11.001001, where the . represents the decimal place.
+
+    We can specify a fixed point real number by the tuple bitsize, num_frac and
+    signed, with num_int determined as (bitsize - num_frac - n_sign).
 
     Attributes:
-        int_bitsize: The number of qubits used to represent the integer part of the float.
-        frac_bitsize: The number of qubits used to represent the fractional part of the float.
+        bitsize: The number of qubits used to represent the fixed point real number.
```

Comment:
nit: Update docstring. `bitsize` now corresponds to total number of bits for integer and fractional part combined.

Reply:
Done.